ReleaseNotes.md

```diff
@@ -18,6 +18,72 @@ Release Notes
 OpenRocket 22.02 (2023-02-XX)
 ------------------------
 
+The 22.02 release includes hundreds of new features, bug fixes, and UI improvements, more than we could ever fit into one set of release notes.  These notes summarize the highlights of the entire release; for more detail consult the notes from the five previous public beta releases.
+ 
+Please note that version 22.02 is required for Macs running MacOS 13.0 or later.
+ 
+## New Features
+* Rocket design features
+  * Pods and strap-on boosters
+  * Tail cones for pods
+  * Freeform fins on nose cones and transitions
+  * Highly configurable rail buttons
+  * Massive update to parachute configuration and part library
+  * Dave Cook's extensive component library now built-in
+  * Coefficient of Drag Override for components and assemblies (and more flexible override settings in general)
+  * Lots more detailed geometry warnings which identify the components involved
+* Appearance updates
+  * Settable opacity per-component
+  * Separate inner/outer appearance for tubes
+  * Separate left/right appearance for fins
+* App Infrastructure
+  * Packaged installers for Windows, Linux, and Mac (JAR file still available)
+  * Updated to Java11
+  * Native ARM version for Apple Silicon Macs (snappy!)
+  * File association for ORK files: double-click files to open in the app
+
+## Simulation and Staging Improvements
+  * Assorted CP calculation fixes
+  * Improved nose cone drag calculations
+  * Improved ground hit velocity calculations
+  * Completely reworked tube fin simulation (no longer "experimental")
+  * Improved time step selection for descent
+  * Simulations now properly account for disabled stages
+  * Update CG correctly when stages are toggled on or off
+  * Displaly stage names in stage enable buttons
+  * More reliable deployment of booster stage recovery device
+
+## UI Improvements
+* Updated icons everywhere
+* Rocket figure display updates
+  * Option to display stability margin as percentage of rocket length
+  * New "Top View" option
+  * Cleaned-up toolbar
+  * Option to show/hide warnings
+  * Option to show/hide CG and CP markings 
+  * Improved click and double-click behavior on rocket
+* Component Configuration Editor
+  * Extensively reorganized for clarity and consistency
+  * Massively improved keyboard navigation
+  * Config panels remember which tab you were on previously
+  * Cancel button to exit config panel without saving changes
+* Contextual menus for managing the component tree
+* Simultaneous multi-component edit (great for appearance editing)
+* Completely updated operation of motor config and simulation tables
+* Improved workflow when creating motor configurations
```

Comment:
Maybe add something about the improved simulation plot? Because we finally got rid of that ugly grey background...

Reply:
I'll believe it if you tell me... I can't remember what it used to look like. :)